### ISSUE TITLE
Mismatched branch flow rate can cause incorrect outdoor air flow rates

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5119]
Helpdesk ticket 10685, 
User file had autosized branch flow rate, but hard flow rates for other system flows.  This resulted in higher OA flow rates than expected, because the branch autosized to 1.32 but the system flow elsewhere was 4.44.  

This is directly related to #3001 which added a check to warn if the branch flow rate was _higher_ than the system flow rate, but there is not check for the reverse.  Ultimately, it does not make sense to use the branch flow rate for anything, especially not for outdoor air calculations.

Related to #4633 and possibly #4568 .
- [ ] Remove the dependency of OA flow rates on branch max flow rate.
- [ ] If possible, deprecate Branch object Maximum Flow Rate {m3/s} field entirely.  May have to push this to a new followup issue.  This field is already unused for plant loops.  Should be unused for airloops as well.  And when this field is removed, suggest removing all the node names from the branch object as well - for now, pobably just deprecate the flow field, but leave it in place.  Can change the Branch object in the next release.
- [ ] Can AirloopHVAC object Design Supply Air Flow Rate {m3/s} be deprecated as well?  OR made to be king?  Figure out where it is used.

**Comment by zhukovgreen** [comment]
@mjwitte Thank you for the explanation

**Comment by mjwitte** [comment]
Preliminary notes.  In [SimAirServingZones](https://github.com/NREL/EnergyPlus/blob/2d46c7212f35cfcf2fed462d692598b3ff5da8a2/src/EnergyPlus/SimAirServingZones.cc#L2008-L2009), the branch max massflow is used in several places to initialize flows in the loop.

```
                    Node( NodeNumIn ).MassFlowRate = PrimaryAirSystem( AirLoopNum ).Branch( InBranchNum ).MaxMassFlowRate;
                    AirLoopFlow( AirLoopNum ).DesSupply = PrimaryAirSystem( AirLoopNum ).Branch( InBranchNum ).MaxMassFlowRate;
```

AirLoopFlow( AirLoopNum ).DesSupply is used heavily in MixedAir::CalcOAController
_One option here is to use the AirLoopHVAC Design Supply Air Flow Rate instead._  Would this work for dualduct systems?  Or other configurations which may not simply has a single design flow rate?

**Comment by Myoldmopar** [comment]
Closed via #5582
